src/modules/SdnDiag.Utilities.Config.psd1

```diff
@@ -8,4 +8,16 @@
     )
     DefaultModuleDirectory = "C:\Program Files\WindowsPowerShell\Modules\SdnDiagnostics"
     WorkingDirectory = "C:\Windows\Tracing\SdnDiag"
+    ExportFileJsonDepth = @{
+        3 = @(
+            'Get-NetAdapterChecksumOffload'
+            'Get-NetAdapterLso'
+            'Get-NetAdapterRdma'
+            'Get-NetAdapterRsc'
+            'Get-NetAdapterRss'
+            'Get-NetAdapterStatistics'
+            'Get-NetIPConfiguration'
+            'Get-SdnVfpVmSwitchPort'
```

Comment:
>Get-SdnVfpVmSwitchPort [](http://example.com/codeflow?start=13&length=22)

Missing: NetAdapterBinding

Reply:
@sbgms from what I am seeing in results of the json file, do not see need to expand 3 levels.